Build fails on armv7 FreeBSD 13

Building the port `sysutils/nomad-pot-driver` on armv7 FreeBSD 13 fails like this:

```
===>  Building nomad-pot-driver from 
internal/unsafeheader
internal/abi
internal/cpu
... snip ...
github.com/shirou/gopsutil/cpu
github.com/shirou/gopsutil/disk
# github.com/shirou/gopsutil/cpu
vendor/github.com/shirou/gopsutil/cpu/cpu_freebsd.go:25:16: undefined: cpuTimes
vendor/github.com/shirou/gopsutil/cpu/cpu_freebsd.go:42:31: undefined: cpuTimes
vendor/github.com/shirou/gopsutil/cpu/cpu_freebsd.go:66:38: undefined: cpuTimes
vendor/github.com/shirou/gopsutil/cpu/cpu_freebsd.go:72:15: undefined: cpuTimes
vendor/github.com/shirou/gopsutil/cpu/cpu_freebsd.go:87:13: undefined: cpuTimes
github.com/shirou/gopsutil/mem
# github.com/shirou/gopsutil/disk
vendor/github.com/shirou/gopsutil/disk/disk_freebsd.go:26:36: undefined: MNT_WAIT
vendor/github.com/shirou/gopsutil/disk/disk_freebsd.go:31:15: undefined: Statfs
vendor/github.com/shirou/gopsutil/disk/disk_freebsd.go:32:25: undefined: MNT_WAIT
vendor/github.com/shirou/gopsutil/disk/disk_freebsd.go:36:17: undefined: MNT_RDONLY
vendor/github.com/shirou/gopsutil/disk/disk_freebsd.go:39:17: undefined: MNT_SYNCHRONOUS
vendor/github.com/shirou/gopsutil/disk/disk_freebsd.go:42:17: undefined: MNT_NOEXEC
vendor/github.com/shirou/gopsutil/disk/disk_freebsd.go:152:9: undefined: Bintime
vendor/github.com/shirou/gopsutil/disk/disk_freebsd.go:161:22: undefined: Statfs
vendor/github.com/shirou/gopsutil/disk/disk_freebsd.go:165:54: undefined: Statfs
vendor/github.com/shirou/gopsutil/disk/disk_freebsd.go:180:32: undefined: Devstat
vendor/github.com/shirou/gopsutil/disk/disk_freebsd.go:42:17: too many errors
... snip ...
```

These issues are [well known](https://github.com/shirou/gopsutil/issues/627) and already fixed upstream.  You just need to bump the requested version of dependency `github.com/shirou/gopsutil` to make the build succeed.

--- a/go.mod
+++ b/go.mod
@@ -26,7 +26,6 @@ require (
 	github.com/hashicorp/go-envparse v0.0.0-20190703193109-150b3a2a4611 // indirect
 	github.com/hashicorp/go-getter v1.4.0 // indirect
 	github.com/hashicorp/go-hclog v0.10.0
-	github.com/hashicorp/go-plugin v1.0.1 // indirect
 	github.com/hashicorp/hcl2 v0.0.0-20191002203319-fb75b3253c80 // indirect
 	github.com/hashicorp/nomad v0.10.1
 	github.com/hashicorp/nomad/api v0.0.0-20191203164002-b31573ae7206 // indirect
@@ -36,7 +35,7 @@ require (
 	github.com/opencontainers/runc v1.0.0-rc8.0.20190611121236-6cc515888830 // indirect
 	github.com/opencontainers/selinux v1.3.1 // indirect
 	github.com/seccomp/libseccomp-golang v0.9.1 // indirect
-	github.com/shirou/gopsutil v2.19.11+incompatible // indirect
+	github.com/shirou/gopsutil v3.21.10+incompatible // indirect
 	github.com/syndtr/gocapability v0.0.0-20180916011248-d98352740cb2 // indirect
 	github.com/ugorji/go v1.1.7 // indirect
 	github.com/vbatts/tar-split v0.11.1 // indirect

